README.md

```diff
@@ -18,16 +18,225 @@ To reconstruct the SIM data and create OTFs from PSFs from the cryoSIM at B24, D
+
+Overrides:
+  Arguments that override configured values. Defaults stated are only used
+  if no value is given or configured.
+
+  --ndirs NDIRS         Number of SIM directions
+  --nphases NPHASES     Number of pattern phases per SIM direction
+  --nordersout NORDERSOUT
+                        Number of output SIM orders (must be <= nphases//2;
+                        safe to ignore usually)
+  --angle0 ANGLE0       Angle of the first SIM angle in radians
+  --ls LS               Line spacing of SIM pattern in microns
+  --na NA               Detection objective's numerical aperture
+  --nimm NIMM           Refractive index of immersion medium
+  --wiener WIENER       Wiener constant; lower value leads to higher
+                        resolution and noise; playing with it extensively is
+                        strongly encouraged
+  --zoomfact ZOOMFACT   Lateral zoom factor in the output over the input
+                        images; leaving it at 2 should be fine in most cases
+  --zzoom ZZOOM         Axial zoom factor; almost never needed
+  --background BACKGROUND
+                        Camera readout background
+  --usecorr USECORR     Use a flat-field correction file if provided
+  --k0angles K0ANGLES K0ANGLES K0ANGLES
+                        Use these pattern vector k0 angles for all directions
+                        (instead of inferring the rest angles from angle0)
+  --otfRA OTFRA         Use rotationally averaged OTF; otherwise uses 3/2D OTF
+                        for 3/2D raw data
+  --otfPerAngle OTFPERANGLE
+                        Use one OTF per SIM angle; otherwise one OTF is used
+                        for all angles, which is how it's been done
+                        traditionally
+  --fastSI FASTSI       SIM image is organized in Z->Angle->Phase order;
+                        otherwise assuming Angle->Z->Phase image order
+  --k0searchAll K0SEARCHALL
+                        Search for k0 at all time points
+  --norescale NORESCALE
+                        No bleach correction
+  --equalizez EQUALIZEZ
+                        Bleach correction for z
+  --equalizet EQUALIZET
+                        Bleach correction for time
+  --dampenOrder0 DAMPENORDER0
+                        Dampen order-0 in final assembly; do not use for 2D
+                        SIM; good choice for high-background images
+  --nosuppress NOSUPPRESS
+                        Do not suppress DC singularity in the result (good
+                        choice for 2D/TIRF data)
+  --nokz0 NOKZ0         Do not use kz=0 plane of the 0th order in the final
+                        assembly (mostly for debug)
+  --gammaApo GAMMAAPO   Output apodization gamma; 1.0 means triangular apo;
+                        lower value means less dampening of high-resolution
+                        info at the trade-off of higher noise
+  --explodefact EXPLODEFACT
+                        Artificially exploding the reciprocal-space distance
```

Comment:
Tense

Reply:
This was already "Artificially explode"

Reply:
Ah, I was looking at settings/formatting.py